test/types/test.ts

```diff
@@ -358,6 +359,15 @@ eventBus.once('foo', 2000, callback);
 
 eventBus.once('foo', callback, this);
 
+const foo = diagram.invoke((modeling: Modeling, eventBus: EventBus) => {
```

Comment:
Already verified in line 145.

Reply:
No. This one specifically verifies the return semantics.